### PR TITLE
Nicer tutorial

### PR DESCRIPTION
**Pull request by steven-murray**
Adds a tutorial following the step-by-step outline in Liu+Shaw (2020).

The idea is to introduce the idea behind 21cmSense, not so much the code itself (but with worked examples from 21cmSense code).

### CODE REVIEW

**Comment by steven-murray** [comment]
@jpober would be great if you could have a read through the tutorial and make sure I understand things aright (and explain them right). The weakest part is I think the explanation for the time-evolution of the UVWs.

File: docs/tutorials/understanding_21cmsense.ipynb [hunk 1]
```diff
@@ -0,0 +1,1197 @@
+{
```
Comment (mkolopanis) [inline docs/tutorials/understanding_21cmsense.ipynb:1]: <p>The formula for final averaged errorbars doesn't have the correct units. should it be 1/sqrt(sum(eppsilon^-2))?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/steven-murray/21cmSense/pull/44/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Comment (mkolopanis) [inline docs/tutorials/understanding_21cmsense.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #4.        hera.baselines_metres[:,:, 1].flatten());
</pre><p>the spacing here compared to the previous line looks odd.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/steven-murray/21cmSense/pull/44/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Comment (steven-murray) [reply docs/tutorials/understanding_21cmsense.ipynb:1]: <p>Yes.</p>

Comment (steven-murray) [reply docs/tutorials/understanding_21cmsense.ipynb:1]: <p>So it does. Fixed.</p>

**Comment by steven-murray** [comment]
@mkolopanis thanks for the comments, I've addressed them!

**Review by mkolopanis — APPROVED** [review]
This looks good to me. Not sure if @jpober wants to look through it first or not so I'll approve but not merge for now.

**Comment by jpober** [comment]
Don't wait for me.  I'll look eventually but not any time this year :(

**Pull request merged**